unicodetools/data/emoji/dev/internal/emojiOrdering.txt

```diff
@@ -0,0 +1,341 @@
+@@ Smileys_and_Emotion
+@ face-smiling
+😀 😃 😄 😁 😆 😅 🤣 😂
+🙂 🙃 🫠 😉 😊 😇
```

Comment:
Why are you adding this file here?
Duplicate of unicodetools/src/main/resources/org/unicode/tools/emoji/emojiOrdering.txt ?

Reply:
Please see my latest comment in #310.

Reply:
Looking at the history, Ned removed this file in PR #276, and in issue #283 we decided to keep it that way.
Please don't add it back again.

Reply:
OK, I will keep it in Generated for the time being.

Reply:
Great, please remove this file from this PR.
“All checks have passed” but let me know if you need more help.

Reply:
[2f2a360](https://github.com/unicode-org/unicodetools/pull/397/commits/2f2a360e50334419c438dd4a4eeb448bce379597)